versions.cfg

```diff
@@ -325,3 +327,11 @@ plone.app.drafts                      = 1.1.1
 plone.app.tiles                       = 2.2.1
 plone.tiles                           = 1.8.0
 plone.app.ldap                        = 1.4.0
+
+[versionannotations]
+# keep this alphabetical please
```

Comment:
Keeping it alphabetical seems very good. Can you start by moving zope.component above zope.interface? :-)
Someone can also do that manually before merging, if the pull request is good.